core/src/com/google/inject/internal/package-info.java

```diff
@@ -17,5 +17,4 @@
 /**
  * <i>Guice</i> (sounds like like "juice")
  */
-
-package com.google.inject.internal;
```

Comment:
It's staying. You can look at the full file at this revision here: https://github.com/google/guice/blob/merge-internal-changes/core/src/com/google/inject/internal/package-info.java